idea/idea-frontend-api/src/org/jetbrains/kotlin/idea/frontend/api/components/KtSymbolDeclarationRendererProvider.kt

```diff
@@ -24,6 +25,8 @@ data class KtTypeRendererOptions(
      * @sample Function0<Int> returns () -> Int
      */
     val renderFunctionType: Boolean = true,
+
+    val renderAnnotations: Boolean = true,
```

Comment:
There is already `RendererModifier.ANNOTATIONS` option available in `KtDeclarationRendererOptions.modifiiers`. May we reuse it?

Reply:
Actually it looks like we don't need it. We will need more sophisticated annotation filter instead.